plugins/woocommerce-admin/client/products/fields/attribute-field/attribute-field.tsx

```diff
@@ -50,5 +59,88 @@ export const AttributeField: React.FC< AttributeFieldProps > = ( {
 			</div>
 		);
 	}
-	return <div className="woocommerce-attribute-field"></div>;
+
+	const sortedAttributes = value.sort( ( a, b ) => a.position - b.position );
+	const attributeKeyValues = value.reduce(
+		(
+			keyValue: Record< number, ProductAttribute >,
+			attribute: ProductAttribute
+		) => {
+			keyValue[ attribute.id ] = attribute;
+			return keyValue;
+		},
+		{} as Record< number, ProductAttribute >
+	);
+	return (
+		<div className="woocommerce-attribute-field">
+			<Sortable
+				onOrderChange={ ( items ) => {
```

Comment:
Side note: looking for feedback from you and @joelclimbsthings on this API so far.  Does the simplicity of `onOrderChange` make sense to you?  Is there something the `Sortable` component could do to make things easier for consumers?

Reply:
The one thing I was thinking of, was for the callback to provide the actual data type versus the JSX.Element, so we don't have to add extra logic on getting the key and such.
Maybe this can be passed in as an optional data type of the nested element, or we pass in the list and it matches the `key` for us on the Sortable list end?

Reply:
So you're thinking this would return an array of the newly ordered keys?

I had considered something similar, but seems like it would be more work on the consumer to then order their items, match with the correct elements, and store in state.

Or did you have a slightly different process in mind?

Reply:
> So you're thinking this would return an array of the newly ordered keys?

That would work, but I was more so thinking we pass back the actual objects that match the keys.
In the case of attributes for example, we also pass in the actual attribute list, then the callback returns to us the re-ordered attribute list that match the keys.
But maybe this will add to much logic to the `Sortable` component.

If we just went the key route, it would prevent us from having to filter out any `undefined` keys, as the `key` can be undefined on a JSX.Element type.